class CallException(Exception):
    pass


class RegisterException(Exception):
    pass
